circle/circle.go

```diff
@@ -17,7 +18,12 @@ Square: 201.06192982974676
 */
 func main() {
 	fmt.Println("The program prints the perimeter and the square of a circle given the radius.")
-	// ++ Your code here! ++
-	//
-	// Hint: Use math.Pi constant (https://pkg.go.dev/math#pkg-constants) to obtain π value.
+	fmt.Println("Enter the radius: ")
+	var r float64
+	fmt.Scan(&r)
+	Perimeter := 2 * math.Pi * r
```

Comment:
nit: please use `mixedCase` naming for local variables https://en.wikipedia.org/wiki/Naming_convention_(programming)#Go 

nit: in this case it could be just `p` (or `s` instead of `square`)